### DISCUSSION TITLE
v2.0.2

### DISCUSSION THREAD

**Discussion by BrycensRanch** [arklanq/next-recompose-plugins#3] [General]
Bro this bug had me smashing my keyboard wondering why my plugins weren't applying! 

Great work on this new release!

**Comment by arklanq** [comment]
> Bro this bug had me smashing my keyboard wondering why my plugins weren't applying!
> 
> Great work on this new release!

I truly apologize for the issue, although all tests were passing, I haven't tried to test scenario where only one plugin have been applied (as the conception of this package is to compose many plugins). In this release issue has been fixed and tests were greatly improved to prevent regression.

Have a good day.
